README.md

```diff
@@ -29,12 +29,22 @@ Before getting started, make sure you have the following prerequisites installed
 
 1. [Node.js 8 or later](https://nodejs.org/en/download/) or using [nvm](https://github.com/nvm-sh/nvm#installation-and-update)
 2. Your IDE of choice
-3. [Docker for Desktop](https://www.docker.com/products/docker-desktop)
-    - If you're on Linux you should use Minikube. Follow the instructions at https://kubernetes.io/docs/tasks/tools/install-minikube/. 
+3. ***On Mac or Windows***: [Docker for Desktop](https://www.docker.com/products/docker-desktop)
+
+
+***On Linux***: Docker for Desktop is not available, alternatives are:
+- [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
```

Comment:
microk8s has a few more steps in this doc, so may _appear_ more complex than minikube... except that the `docker push` is usually mandatory for Kube, so omitting it might not be helpful to someone trying to redo this on IBM cloud (for example). Also, the one liner "install minikube" omits that installing vbox is 4 lines, installing kubectl is 4 lines, etc. Note that snap is already suggested for Helm, below.

Its 6 of one, half a dozen of the other. I am `+0` on microk8s for its ease of removal and resetting, and it doesn't have the weird "point your local docker command into a VM step", which makes `docker run` a bit hard to use.

Reply:
Keep both as options for Linux - and we'll hide some of the microk8s/minikube steps later on